tests/components/primitive-components/invalid-jsx.test.tsx

```diff
@@ -0,0 +1,14 @@
+import { test, expect } from "bun:test"
+import { Circuit } from "lib"
+
+test("snippet-import1-contribution-board", async () => {
+  const circuit = new Circuit()
+
+  expect(() => {
+    circuit.add(
+      <board width="10mm" height="10mm">
+        {1}
+      </board>,
+    )
+  }).toThrowError("Invalid JSX Element")
+})
```

Comment:
make sure to test with `new ClassName()` because i think JSON.stringify could error

Reply:
or you could put in a function or something